pkg/sdk/testint/event_tables_integration_test.go

```diff
@@ -0,0 +1,227 @@
+package testint
+
```

Comment:
There is a row access policy test missing here. You can use `alter view: add and drop row access policies` from `views_gen_integration_test.go`.

Reply:
done